omod/src/main/java/org/openmrs/web/dwr/DWRPatientService.java

```diff
@@ -117,7 +118,10 @@ public static void setMaximumResults(Integer maximumResults) {
 		
 		patientList = new Vector<Object>(patients.size());
 		for (Patient p : patients) {
-			patientList.add(new PatientListItem(p, searchValue));
+			PatientListItem htmlSafePatient = new PatientListItem(p, searchValue);
+			htmlSafePatient.setGivenName(WebUtil.escapeHTML(htmlSafePatient.getGivenName()));
```

Comment:
Thanks @The-Lady , hi @isears  are you fine with getting things from the same object and putting them back?

Reply:
@HerbertYiga I am sorry I forgot it is not a good practice to make changes in the objects this way. If preferred, I can make a clone of the object and make changes to that before adding it to the list.

Reply:
That could be good. Also maybe change `htmlSafePatient` to `htmlSafePatientLI` or something to make it a bit more readable and make it clear that it's a list item object not a patient object.

Reply:
@isears Made the desired changes. Thanks for reviewing. Let me know if any other changes are required.